src/components/SearchBar.vue

```diff
@@ -78,6 +89,15 @@ export default {
     },
     goToProfile() {
       this.$router.push({ name: "CreateUserProfile" });
+    },
+    computedSearch(search) {
+      this.suggested = search;
+      return search;
+    },
+    onOptions(options) {
```

Comment:
I think I need to speak to you directly about how to go about this the right way. Just realized I might be doing things in a very bad way.

Reply:
Actually, I've decided to keep this as is for the time being, since it's working. I've tacked on my request and have subscribed to [this issue](https://github.com/sagalbot/vue-select/issues/816) on Github so I'll be notified if their position changes on the matter, so I think we can proceed with the code as-is.